test/e2e/image_sign_test.go

```diff
@@ -26,10 +26,17 @@ var _ = Describe("Podman image sign", func() {
 	})
 
 	AfterEach(func() {
+		// There's no way to run gpg without an agent, so, clean up
+		// after every test. No need to check error status.
+		cmd := exec.Command("gpgconf", "--kill", "gpg-agent")
+		cmd.Stdout = GinkgoWriter
+		cmd.Stderr = GinkgoWriter
+		_ = cmd.Run()
+
 		os.Setenv("GNUPGHOME", origGNUPGHOME)
 	})
 
-	It("podman sign image", func() {
+	It("podman sign image", Serial, func() {
```

Comment:
The Serial decorators can be added to the Describe() function, this would be better than having to set it for each It() block. Considering that people could add new tests here and may not pay attention to such details